src/components/Organisms/MarketingPreferencesDS/_MarketingPrefsConfig.js

```diff
@@ -0,0 +1,144 @@
+import * as yup from 'yup';
+import 'yup-phone';
+import { merge } from 'lodash';
+
+const setInitialValues = overrideValues => {
+  const defaultValues = {
+    mp_email: '',
+    mp_mobile: '',
+    mp_phone: '',
+    mp_address1: '',
+    mp_address2: '',
+    mp_address3: '',
+    mp_town: '',
+    mp_country: '',
+    mp_postcode: '',
+    mp_permissionEmail: [],
+    mp_permissionPost: [],
+    mp_permissionPhone: [],
+    mp_permissionSMS: []
+  };
+
+  // Override with any custom options supplied
+  const updatedValues = merge(defaultValues, overrideValues);
+
+  return updatedValues;
+};
+
+const buildValidationSchema = overrideOptions => {
```

Comment:
I somehow completely missed this file on first look, & it helps a lot with filling in some of the blanks re config!